packages/app/src/navigation/SidebarNavigationHeader.vue

```diff
@@ -87,7 +99,7 @@ useSubscription({ query: SidebarNavigationHeaderBranchChangeDocument })
 const showModal = ref(false)
 
 const props = defineProps<{
-  gql: SidebarNavigationHeaderFragment
+  gql?: SidebarNavigationHeaderFragment
```

Comment:
Yes, in this case it can be undefined. Ultimately it comes from `SidebarNavigationContainer` where we pass
```js
:gql="query.data.value"
```
I think that's why it can be undefined

<img width="825" alt="Screenshot 2023-06-07 at 3 19 31 PM" src="https://github.com/cypress-io/cypress/assets/14275198/eeaeb505-f0c7-4d0c-b39b-7db60b7e21b2">